README.md

```diff
@@ -119,42 +119,52 @@ All designs are in [Figma](https://www.figma.com/files/team/834568130405102661/H
   - ```nvm use `cat .nvmrc` ```
 
 Optional:
-- `VSCode`, with `eslint` and `prettier` plugins
+- `VS Code`, with [recommended extensions](./.vscode/extensions.json).
+  - While this is optional, it is worth noting that it is in use by the whole development team
+  - Using VS Code lets us share [default configuration settings](./.vscode/default.settings.json) and easily run scripts using [VS Code tasks](./.vscode/tasks.json).
 - `docker`
+  - Docker is necessary in order to create images for the [deploy process](https://hms-dbmi.atlassian.net/wiki/spaces/GL/pages/3009282049/Deployment)
+  - It is also used to run a local instance of the application when using the test scripts in the `./etc` directory
 
 ### Development
 
 After checking out the project, cd-ing into it, and setting up a Python 3.9 virtual environment,
-- Get `app.conf` from another developer and place it at `context/instance/app.conf`.
+- Get `app.conf` from [Confluence](https://hms-dbmi.atlassian.net/wiki/spaces/GL/pages/3045457929/app.conf) or from another developer and place it at `context/instance/app.conf`.
```

Comment:
Should we consider moving the app.conf to 1Password? Doesn't necessarily have to be in the lab vault; could just in in a developer's vault and shared to new devs during onboarding.

Reply:
I think that would be a great idea; however, it seems like we can't use 1Password's document storage functionality:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/67d1c015-c4ad-4c17-8fb6-fe61649882ba)

We could potentially use the `secure note` function for this since it's plaintext.

I've created a ticket to track this: https://hms-dbmi.atlassian.net/browse/HMP-384